src/main/java/hu/oe/nik/szfmv/automatedcar/model/Transform.java

```diff
@@ -23,14 +44,14 @@ public Transform(double m11, double m12, double m21, double m22){
     }
 
     public Transform(){
-        this.m11 = 0;
-        this.m12 = 0;
-        this.m21 = 0;
-        this.m22 = 0;
+        this.m11 = 0.0;
+        this.m12 = 0.0;
+        this.m21 = 0.0;
+        this.m22 = 0.0;
         this.rotation = this.calculateRotation();
```

Comment:
ModelCommonUtil.calcRotation

Reply:
Ezeket a konstruktorokat eliminálni lehet majd mindenhonnan, mert mindent XML-ből olvasunk majd be. Ezt most nem tenném meg, mert majd a WorldManager megírása után törölhetők, abban a branchben célszerű refaktorálni és ahol konstruktor hívás volt, le kell cserélni a WM megfelelő metódusára.